spring-cloud-gcp-core/src/main/java/org/springframework/cloud/gcp/core/DefaultCredentialsProvider.java

```diff
@@ -125,7 +125,7 @@ else if (credentials instanceof ComputeEngineCredentials) {
 			}
 		}
 		catch (IOException ioe) {
-			LOGGER.error("No credentials were found.", ioe);
+			LOGGER.error("No core credentials are set.");
```

Comment:
Won't it help to extend this message to say what is going to happen then?
For example mention those credentials in the target module.

I mean that always was good experience for me working with Spring that it has all those good explanations in the logs when we have this or that warning.

Actually why is this `error`, but not `warn` if this isn't going to be fatal for the app?